feat: adding footer

Srish-ty opened the pull request:
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [x] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

Lucif3r-in commented:
Resolve issues @Srish-ty

Lucif3r-in commented on src/components/Footer/Footer.jsx:
```diff
@@ -0,0 +1,45 @@
+      </FirstRow>
+
+      <AboutBox>
+        <LogoCont> <LogoImg src={NitLogo} alt="nit logo" /> </LogoCont>
+        <Para>
+          <Heading3>About NIT Rourkela</Heading3>
+          <Content>
+          With a strength of around 7000 students, NIT Rourkela is one of the most 
+          coveted institutes for higher studies by future technocrats. With its lush green
+          and humongous area, it indeed makes a pleasant campus impression.
+          </Content>
```
Move these to config/content

Lucif3r-in commented:
Do not make redundant commit messages.

Lucif3r-in commented:
@Srish-ty Check for checks failing

Lucif3r-in commented on src/components/Footer/AboutBox.jsx:
```diff
@@ -0,0 +1,96 @@
+
+export const AboutBox = styled.div`
+  ${tw`
+    flex
+    flex-row
+    text-lg
+    w-[851px]
+    mx-4
+    my-8
+    p-5
+    `}
+  background: #080822;
+`;
+
+export const Para = styled.p`
+  ${tw`
+    flex
+    flex-col
+    mx-5
+    
+    `}
+`;
+export const Content = styled.p`
+  ${tw`
+    my-5
+    text-neutral-white-700
+    `}
+`;
+
+export const LogoCont = styled.span`
+  ${tw`
+  w-[250px]
+`}
+`;
+
+export const LogoImg = styled.img`
+  ${tw`
+    w-full
+    `}
+`;
```
Move all the styles to styles.js in this directory

Srish-ty replied:
done [commit](https://github.com/dscnitrourkela/project-oregano/pull/241/commits/b5fe63a6079822c58f7e5b74b4f79047342b44fal)

Lucif3r-in commented on src/components/Footer/Final_pend.jsx:
Change the file name to something meaningful and make sure they are in CamelCase

Srish-ty replied:
done. [commit link](https://github.com/dscnitrourkela/project-oregano/pull/241/commits/d32684d4bb56baac87a9e0c4f7cf03d612378d8e)

Lucif3r-in commented on src/components/Footer/Final_pend.jsx:
```diff
@@ -0,0 +1,86 @@
+    left: 19px;
+    background: #bdc3c7;
+  }
+`;
```
Move the styles to `styles.js`

Srish-ty replied:
moved. [commited](https://github.com/dscnitrourkela/project-oregano/pull/241/commits/b5fe63a6079822c58f7e5b74b4f79047342b44fa)

Lucif3r-in commented on src/components/Footer/Final_pend.jsx:
```diff
@@ -0,0 +1,86 @@
+  return (
+    <CradleContainer>
+      <CradleBefore></CradleBefore>
+      <Ball1></Ball1>
+      <Ball></Ball>
+      <Ball></Ball>
+      <Ball></Ball>
+      <Ball5></Ball5>
```
Map these `Ball` elements as we may expand the number of socials later

Srish-ty replied:
done. imported socials array from config and mapped. [commit link](https://github.com/dscnitrourkela/project-oregano/pull/241/commits/d8c84cbc76fde7e480f3bfe1ce508a9e7eaaec11)

Lucif3r-in commented on src/components/Footer/Footer.jsx:
```diff
@@ -0,0 +1,27 @@
+      <FooterContainer>
+        <InitialCont>
+          <AboutContainer AbtLogo={NitLogo} about_content={footer.cards[0]} />
+
```
Remove any extra space in each and every file

Srish-ty replied:
removed.

Lucif3r-in commented on src/components/Footer/cradle.styles.css:
Remove this file

Lucif3r-in commented:
@Srish-ty use cloudinary for storing images. Do not use local storage

Srish-ty commented:
> @Srish-ty use cloudinary for storing images. Do not use local storage

removed images and added cloudinary links  [commited changes link](https://github.com/dscnitrourkela/project-oregano/pull/241/commits/ca9b80e290331a2a1af18dfea4560c99a6d8da27)

CIPHERTron commented on src/components/marginals/Footer/Footer.jsx:
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import footer from '../../../../config/content/Footer';
```
Would suggest using absolute instead of relative imports :)

Srish-ty replied:
done. [commit link](https://github.com/dscnitrourkela/project-oregano/pull/241/commits/a3cb7fccba0cdf4019181b48074cb7b64d280606)

CIPHERTron commented on src/components/marginals/Footer/AboutBox.jsx:
```diff
@@ -0,0 +1,37 @@
+  return (
+    <FirstAboutContainer>
+      <FirstRow>
+        <FrItems>About US</FrItems>
```
Follow DRY principle as much as possible.
In cases like these, where you have to use the same component multiple times, run it through a map and make the component dynamic with props.

Srish-ty replied:
made changes. [commited](https://github.com/dscnitrourkela/project-oregano/pull/241/commits/d0620ff74ed2966e2f08b672b905aa4df4b88997)

CIPHERTron reviewed (commented):
Just a couple of suggestions to everyone that I have mentioned below and if you are using linting tools, make sure you adhere to the rules you have defined.

Instead of disabling eslint errors, try to debug them. It'll help you learn a lot about writing clean and modular code
